docs/guide/instruction_tuning.md

```diff
@@ -0,0 +1,90 @@
+# Instruction finetuning
+
+This tutorial will guide you through the basics of instruction finetuning using the Megatron-LLM codebase, using LLaMa 2 7B as the base network.
```

Comment:
add that these same scripts will also work for larger models such as the 70b llama or falcon

Reply:
Done